packages/storybook-react/stories/Usage.mdx

```diff
@@ -14,33 +14,21 @@ First install the React package.
 npm i @uqds/react
 ```
 
-## CSS
+## Global CSS
 
-CSS styles for all components are bundled into a single CSS import you'll need to add to your
-project globally.
+Include Global CSS once for a whole project. Component CSS is handled with
+CSS Modules.
 
 ### PostCSS
 
 ```css
 @import "@uqds/react";
 ```
 
-### Sass
-
-```css
-@use "@uqds/react/src/scss/main" as react;
-```
-
 ### JS
 
 ```js
-import "design-system/packages/react/dist/css/main.css";
-```
-
-Alternatively individual components can be imported from the component packages, for example.
-
-```css
-@import "@uqds/icon";
+import "@uqds/react/dist/main.css";
```

Comment:
Also, can that one use a named export to avoid `dist` appearing in the path?  For example in `package.json`:
```
 "exports": {
    "./css": "./dist/main.css"
  },
```

Then UQDS users can `import "@uqds/react/css";` and they don't need to know where the CSS file actually is or what it's called.